README.md

```diff
@@ -1,3 +1,102 @@
-TuxedoJS
-========
+# TuxedoJS
+
 ![Tux Logo](Tux-full-logo.png "Tux-Logo")
+
+
+> A Front-End Javascript Framework built on Facebook’s powerful React view layer and Flux architecture.
+
+React and Flux are two novel and exciting approaches to front-end development. TuxedoJS implements a more accessible and semantic toolkit that leverages these technologies, abstracting away complexity and extending core functionality. TuxedoJS comes from Tux - a merging of React and Flux.
+
+## Table of Contents
+
+1. [Usage](#usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+    1. [Installing Dependencies](#installing-dependencies)
+    1. [Tasks](#tasks)
+    1. [Roadmap](#roadmap)
+1. [Graceful Degradation](#graceful-degradation)
+1. [Credits and Tech Stack](#credits-and-tech-stack)
+1. [Contributing](#interested-in-contributing)
+1. [Team](#team)
+
+## Usage
+
+TuxedoJS is a modular framework that provides tools for easily rendering components, client-side routing, animating transitions, validating inputs, and managing app data.  Install it via a package manager and include the modules you need to quickly start building your app.
+
+See the full doc site [here](http://www.tuxedojs.org).
+
+## Requirements
+
+- Node 0.10.x
+
+## Development
+
+View the entire [Getting Started]() tutorial to begin hacking with Tux.
+
+### Installing Dependencies
+
+After cloning this repo,  within the root directory:
+
+    npm install tux
+
+### Tasks
+
+Tux is built off of CommonJS and uses a combination of Browserify and Reactify to build client-side dependencies and automatically compile JSX syntax. Because of this, we highly recommend installing the following modules to aid in development while using Tux:
+
+    npm install --save browserify
+    npm install --save envify
+    npm install --save reactify
+    npm install --save watchify
+
+To use these modules in development, we recommend adding the following lines to the `scripts` key of your `package.json` file to set up automatic JSX compiling and configure testing:
+
+    "scripts": {
+      "start": "watchify -d client/app.js -o client/bundle.js -v"
+    }
+
+After adding this code, run
+
+    npm start
+
+during development to automatically compile all JSX and Javascript syntax into one `bundle.js` which you can then directly link to your `index.html` file.
```

Comment:
Add sentence: use `npm run build` to compile a production ready bundle of your code.